src/precompile.jl

```diff
@@ -0,0 +1,32 @@
+let
+    data = load_boston()
+    schema(data)
+    y, X = unpack(data, ==(:MedV))
+    models(matching(X, y))
+    doc("DecisionTreeClassifier", pkg="DecisionTree")
```

Comment:
Lines 5 and 6 test MLJModels.jl functionality; MLJModels is independent of MLJBase